lib/jsonapi_errors_handler.rb

```diff
@@ -39,7 +39,7 @@ def handle_unexpected_error(error)
   end
 
   def render_error(error)
-    render json: ::JsonapiErrorsHandler::ErrorSerializer.new(error), status: error.status
+    render json: ::JsonapiErrorsHandler::ErrorSerializer.new(error), status: error.status.to_i
```

Comment:
The error status can be an integer, that's not an issue. Only in the generated hash, the value should be stringified.